common/config/app.py

```diff
@@ -17,6 +17,7 @@
     default=None,
 )
 
+not_set_base_url = "https://not-set.com"
```

Comment:
@justinorringer Can you update this to `https://example.com` or something similar. `https://not-set.com` is redirecting me to some sketchy site.

Reply:
I didn't even check haha